src/main/java/school/hei/haapi/endpoint/rest/mapper/UserMapper.java

```diff
@@ -90,12 +93,34 @@ public User toDomain(Student student) {
         .lastName(student.getLastName())
         .email(student.getEmail())
         .ref(student.getRef())
-        .status(User.Status.fromValue(student.getStatus().toString()))
+        .status(toDomainUserStatus(student.getStatus()))
         .phone(student.getPhone())
         .entranceDatetime(student.getEntranceDatetime())
         .birthDate(student.getBirthDate())
         .sex(student.getSex() == null ? null : User.Sex.valueOf(student.getSex().toString()))
         .address(student.getAddress())
         .build();
   }
+
+  private User.Status toDomainUserStatus(EnableStatus status) {
+    List<EnableStatus> expectedStatus = Arrays.stream(EnableStatus.values()).toList();
```

Comment:
remove this

Reply:
it's a nice idea, but the current app throws an Exceptiona and shows the current value, not the accepted values whenever faced with the same situation